config/main-page-text.json

```
@@ -0,0 +1,72 @@
+{
+  "WhyDevfiles": {
+    "title": "Why Devfiles?",
```

```suggestion
    "title": "Why devfiles?",
```

I think we prev review suggested we capitalize only if its at the beginning of a sentence. The word is `devfile` in other sentences throughout, so lets stay consistent.

---

Fixed